tests/deprecated_api/test_remove_1-8.py

```diff
@@ -256,3 +269,34 @@ def test_v1_8_0_deprecated_training_type_plugin_property():
     trainer = Trainer()
     with pytest.deprecated_call(match="in v1.6 and will be removed in v1.8"):
         trainer.training_type_plugin
+
+
+@pytest.mark.parametrize(
+    "cls",
+    [
+        DDPPlugin,
+        DDP2Plugin,
+        DDPSpawnPlugin,
+        pytest.param(DeepSpeedPlugin, marks=RunIf(deepspeed=True)),
+        DataParallelPlugin,
+        DDPFullyShardedPlugin,
+        pytest.param(IPUPlugin, marks=RunIf(ipu=True)),
+        DDPShardedPlugin,
+        DDPSpawnShardedPlugin,
+        TPUSpawnPlugin,
+    ],
+)
+def test_v1_8_0_deprecated_training_type_plugin_classes(cls):
+    with pytest.deprecated_call(match="in v1.6 and will be removed in v1.8"):
```

Comment:
Can you check the name is present?

You can access `cls.__name__` and replace "Plugin" -> "Strategy"

Reply:
done